sda-commons-server-key-mgmt/src/main/java/org/sdase/commons/keymgmt/KeyMgmtBundle.java

```diff
@@ -17,6 +17,8 @@
 
 public class KeyMgmtBundle<T extends Configuration> implements ConfiguredBundle<T> {
 
+  public static final String PLACEHOLDER = "PLACEHOLDER";
```

Comment:
Was Stefan involved in the decision for the placeholder value? If not, we have to involve him.

Reply:
I've agreed on the value "UNKNOWN_KEY_VALUE" with Stefan.